--- a/container.if	2020-05-07 13:42:38.164307070 +0200
+++ b/container.if	2020-05-07 14:58:37.616619675 +0200
@@ -525,7 +525,7 @@
     filetrans_pattern($1, container_var_lib_t, container_ro_file_t, dir, "overlay2-images")
     filetrans_pattern($1, container_var_lib_t, container_ro_file_t, dir, "overlay2-layers")
     filetrans_pattern($1, container_var_lib_t, container_ro_file_t, dir, "atomic")
-    userdom_admin_home_dir_filetrans($1, container_home_t, dir, ".container")
+#    userdom_admin_home_dir_filetrans($1, container_home_t, dir, ".container")
     filetrans_pattern($1, container_var_lib_t, container_ro_file_t, dir, "kata-containers")
 
 ')
@@ -567,7 +567,7 @@
 	')
 
 	files_search_pids($1)
-	files_write_all_pid_sockets($1)
+#	files_write_all_pid_sockets($1)
 	allow $1 spc_t:unix_stream_socket connectto;
 ')
 
@@ -801,7 +801,7 @@
 	domain_role_change_exemption($1_t)
 
 	kernel_read_system_state($1_t)
-	kernel_read_all_proc($1_t)
+#	kernel_read_all_proc($1_t)
 
 	mls_file_read_to_clearance($1_t)
 	mls_file_write_to_clearance($1_t)
@@ -837,7 +837,7 @@
 	mcs_constrained($1_t)
 	role system_r types $1_t;
 
-	kernel_read_all_proc($1_t)
+#	kernel_read_all_proc($1_t)
 ')
 
 ########################################
--- a/container.te	2020-05-07 13:42:32.508294149 +0200
+++ b/container.te	2020-05-07 14:56:51.335358894 +0200
@@ -81,8 +81,8 @@
 type container_plugin_var_run_t alias docker_plugin_var_run_t;
 files_pid_file(container_plugin_var_run_t)
 
-type container_unit_file_t alias docker_unit_file_t;
-systemd_unit_file(container_unit_file_t)
+# type container_unit_file_t alias docker_unit_file_t;
+# systemd_unit_file(container_unit_file_t)
 
 type container_devpts_t alias docker_devpts_t;
 term_pty(container_devpts_t)
@@ -155,8 +155,8 @@
 manage_files_pattern(container_runtime_domain, container_home_t, container_home_t)
 manage_dirs_pattern(container_runtime_domain, container_home_t, container_home_t)
 manage_lnk_files_pattern(container_runtime_domain, container_home_t, container_home_t)
-userdom_admin_home_dir_filetrans(container_runtime_domain, container_home_t, dir, ".container")
-userdom_manage_user_home_content(container_runtime_domain)
+# userdom_admin_home_dir_filetrans(container_runtime_domain, container_home_t, dir, ".container")
+# userdom_manage_user_home_content(container_runtime_domain)
 
 manage_dirs_pattern(container_runtime_domain, container_config_t, container_config_t)
 manage_files_pattern(container_runtime_domain, container_config_t, container_config_t)
@@ -226,7 +226,7 @@
 allow container_runtime_domain container_devpts_t:chr_file { relabelfrom rw_chr_file_perms setattr_chr_file_perms };
 term_create_pty(container_runtime_domain, container_devpts_t)
 term_use_all_ttys(container_runtime_domain)
-term_use_all_inherited_terms(container_runtime_domain)
+# term_use_all_inherited_terms(container_runtime_domain)
 
 kernel_read_network_state(container_runtime_domain)
 kernel_read_all_sysctls(container_runtime_domain)
@@ -275,26 +275,26 @@
 files_search_all(container_runtime_domain)
 files_read_usr_symlinks(container_runtime_domain)
 files_search_locks(container_runtime_domain)
-files_dontaudit_unmount_all_mountpoints(container_runtime_domain)
+# files_dontaudit_unmount_all_mountpoints(container_runtime_domain)
 
 fs_read_cgroup_files(container_runtime_domain)
 fs_read_tmpfs_symlinks(container_runtime_domain)
 fs_search_all(container_runtime_domain)
 fs_getattr_all_fs(container_runtime_domain)
-fs_rw_onload_sockets(container_runtime_domain)
+# fs_rw_onload_sockets(container_runtime_domain)
 
 auth_dontaudit_getattr_shadow(container_runtime_domain)
 
 init_read_state(container_runtime_domain)
-init_status(container_runtime_domain)
-init_stop(container_runtime_domain)
-init_start(container_runtime_domain)
-init_manage_config_transient_files(container_runtime_domain)
+# init_status(container_runtime_domain)
+# init_stop(container_runtime_domain)
+# init_start(container_runtime_domain)
+# init_manage_config_transient_files(container_runtime_domain)
 
 logging_send_audit_msgs(container_runtime_domain)
 
 miscfiles_read_localization(container_runtime_domain)
-miscfiles_dontaudit_access_check_cert(container_runtime_domain)
+# miscfiles_dontaudit_access_check_cert(container_runtime_domain)
 miscfiles_dontaudit_setattr_fonts_cache_dirs(container_runtime_domain)
 miscfiles_read_fonts(container_runtime_domain)
 miscfiles_read_hwdata(container_runtime_domain)
@@ -310,9 +310,9 @@
 sysnet_dns_name_resolve(container_runtime_domain)
 sysnet_exec_ifconfig(container_runtime_domain)
 
-optional_policy(`
-	ssh_use_ptys(container_runtime_domain)
-')
+# optional_policy(`
+# 	ssh_use_ptys(container_runtime_domain)
+# ')
 
 optional_policy(`
 	rpm_exec(container_runtime_domain)
@@ -329,14 +329,14 @@
 	iptables_domtrans(container_runtime_domain)
 ')
 
-optional_policy(`
-	openvswitch_stream_connect(container_runtime_domain)
-')
-
-optional_policy(`
-    domain_stub_named_filetrans_domain()
-    container_filetrans_named_content(named_filetrans_domain)
-')
+# optional_policy(`
+# 	openvswitch_stream_connect(container_runtime_domain)
+# ')
+
+# optional_policy(`
+# 	domain_stub_named_filetrans_domain()
+# 	container_filetrans_named_content(named_filetrans_domain)
+# ')
 
 #
 # lxc rules
@@ -359,58 +359,58 @@
 allow container_runtime_domain container_var_lib_t:chr_file mounton;
 can_exec(container_runtime_domain, container_var_lib_t)
 
-kernel_dontaudit_setsched(container_runtime_domain)
+# kernel_dontaudit_setsched(container_runtime_domain)
 kernel_get_sysvipc_info(container_runtime_domain)
 kernel_request_load_module(container_runtime_domain)
-kernel_mounton_messages(container_runtime_domain)
-kernel_mounton_all_proc(container_runtime_domain)
-kernel_mounton_all_sysctls(container_runtime_domain)
+# kernel_mounton_messages(container_runtime_domain)
+# kernel_mounton_all_proc(container_runtime_domain)
+# kernel_mounton_all_sysctls(container_runtime_domain)
 kernel_list_all_proc(container_runtime_domain)
 kernel_read_all_sysctls(container_runtime_domain)
 kernel_rw_net_sysctls(container_runtime_domain)
 kernel_rw_unix_sysctls(container_runtime_domain)
 kernel_dontaudit_search_kernel_sysctl(container_runtime_domain)
-kernel_dontaudit_access_check_proc(container_runtime_domain)
-kernel_dontaudit_setattr_proc_files(container_runtime_domain)
+# kernel_dontaudit_access_check_proc(container_runtime_domain)
+# kernel_dontaudit_setattr_proc_files(container_runtime_domain)
 kernel_dontaudit_setattr_proc_dirs(container_runtime_domain)
-kernel_dontaudit_write_usermodehelper_state(container_runtime_domain)
+# kernel_dontaudit_write_usermodehelper_state(container_runtime_domain)
 
-dev_getattr_all(container_runtime_domain)
-dev_getattr_sysfs_fs(container_runtime_domain)
+# dev_getattr_all(container_runtime_domain)
+# dev_getattr_sysfs_fs(container_runtime_domain)
 dev_read_rand(container_runtime_domain)
 dev_read_urand(container_runtime_domain)
 dev_read_lvm_control(container_runtime_domain)
 dev_rw_sysfs(container_runtime_domain)
 dev_rw_loop_control(container_runtime_domain)
 dev_rw_lvm_control(container_runtime_domain)
-dev_read_mtrr(container_runtime_domain)
+# dev_read_mtrr(container_runtime_domain)
 
-files_getattr_isid_type_dirs(container_runtime_domain)
-files_manage_isid_type_dirs(container_runtime_domain)
-files_manage_isid_type_files(container_runtime_domain)
-files_manage_isid_type_symlinks(container_runtime_domain)
-files_manage_isid_type_chr_files(container_runtime_domain)
-files_manage_isid_type_blk_files(container_runtime_domain)
-files_exec_isid_files(container_runtime_domain)
-files_mounton_isid(container_runtime_domain)
+# files_getattr_isid_type_dirs(container_runtime_domain)
+# files_manage_isid_type_dirs(container_runtime_domain)
+# files_manage_isid_type_files(container_runtime_domain)
+# files_manage_isid_type_symlinks(container_runtime_domain)
+# files_manage_isid_type_chr_files(container_runtime_domain)
+# files_manage_isid_type_blk_files(container_runtime_domain)
+# files_exec_isid_files(container_runtime_domain)
+# files_mounton_isid(container_runtime_domain)
 files_mounton_non_security(container_runtime_domain)
-files_mounton_isid_type_chr_file(container_runtime_domain)
+# files_mounton_isid_type_chr_file(container_runtime_domain)
 
 fs_mount_all_fs(container_runtime_domain)
 fs_unmount_all_fs(container_runtime_domain)
 fs_remount_all_fs(container_runtime_domain)
-files_mounton_isid(container_runtime_domain)
+# files_mounton_isid(container_runtime_domain)
 fs_manage_cgroup_dirs(container_runtime_domain)
 fs_manage_cgroup_files(container_runtime_domain)
-fs_rw_nsfs_files(container_runtime_domain)
+# fs_rw_nsfs_files(container_runtime_domain)
 fs_relabelfrom_xattr_fs(container_runtime_domain)
 fs_relabelfrom_tmpfs(container_runtime_domain)
 fs_read_tmpfs_symlinks(container_runtime_domain)
 fs_list_hugetlbfs(container_runtime_domain)
 fs_getattr_all_fs(container_runtime_domain)
 fs_list_inotifyfs(container_runtime_domain)
-fs_rw_inherited_tmpfs_files(container_runtime_domain)
-fs_read_hugetlbfs_files(container_runtime_domain)
+# fs_rw_inherited_tmpfs_files(container_runtime_domain)
+# fs_read_hugetlbfs_files(container_runtime_domain)
 fs_read_tmpfs_symlinks(container_runtime_domain)
 fs_search_tmpfs(container_runtime_domain)
 fs_rw_hugetlbfs_files(container_runtime_domain)
@@ -420,21 +420,21 @@
 term_use_ptmx(container_runtime_domain)
 term_getattr_pty_fs(container_runtime_domain)
 term_relabel_pty_fs(container_runtime_domain)
-term_mounton_unallocated_ttys(container_runtime_domain)
+# term_mounton_unallocated_ttys(container_runtime_domain)
 
-modutils_domtrans_kmod(container_runtime_domain)
+# modutils_domtrans_kmod(container_runtime_domain)
 
-systemd_status_all_unit_files(container_runtime_domain)
-systemd_start_systemd_services(container_runtime_domain)
+# systemd_status_all_unit_files(container_runtime_domain)
+# systemd_start_systemd_services(container_runtime_domain)
 systemd_dbus_chat_logind(container_runtime_domain)
 systemd_dbus_chat_resolved(container_runtime_domain)
 
-userdom_stream_connect(container_runtime_domain)
+# userdom_stream_connect(container_runtime_domain)
 userdom_search_user_home_content(container_runtime_domain)
 userdom_read_all_users_state(container_runtime_domain)
-userdom_relabel_user_home_files(container_runtime_domain)
-userdom_relabel_user_tmp_files(container_runtime_domain)
-userdom_relabel_user_tmp_dirs(container_runtime_domain)
+# userdom_relabel_user_home_files(container_runtime_domain)
+# userdom_relabel_user_tmp_files(container_runtime_domain)
+# userdom_relabel_user_tmp_dirs(container_runtime_domain)
 userdom_use_inherited_user_terminals(container_runtime_domain)
 userdom_use_user_ptys(container_runtime_domain)
 
@@ -470,7 +470,7 @@
 
 fs_manage_fusefs_dirs(container_runtime_domain)
 fs_manage_fusefs_files(container_runtime_domain)
-fs_manage_fusefs_symlinks(container_runtime_domain)
+# fs_manage_fusefs_symlinks(container_runtime_domain)
 fs_mount_fusefs(container_runtime_domain)
 fs_unmount_fusefs(container_runtime_domain)
 fs_exec_fusefs_files(container_runtime_domain)
@@ -483,8 +483,8 @@
     container_lib_filetrans(container_domain,container_file_t, sock_file)
     container_use_ptys(container_domain)
     container_spc_stream_connect(container_domain)
-    fs_dontaudit_remount_tmpfs(container_domain)
-    dev_dontaudit_mounton_sysfs(container_domain)
+#    fs_dontaudit_remount_tmpfs(container_domain)
+#    dev_dontaudit_mounton_sysfs(container_domain)
     allow container_domain container_file_t:dir_file_class_set { relabelfrom relabelto map };
 ')
 
@@ -498,18 +498,18 @@
 ')
 
 optional_policy(`
-	dbus_system_bus_client(container_runtime_domain)
+#	dbus_system_bus_client(container_runtime_domain)
 	init_dbus_chat(container_runtime_domain)
-	init_start_transient_unit(container_runtime_domain)
+#	init_start_transient_unit(container_runtime_domain)
 
 	optional_policy(`
 		systemd_dbus_chat_logind(container_runtime_domain)
-		systemd_dbus_chat_machined(container_runtime_domain)
+#		systemd_dbus_chat_machined(container_runtime_domain)
 	')
 
-	optional_policy(`
-		dnsmasq_dbus_chat(container_runtime_domain)
-	')
+#	optional_policy(`
+#		dnsmasq_dbus_chat(container_runtime_domain)
+#	')
 
 	optional_policy(`
 		firewalld_dbus_chat(container_runtime_domain)
@@ -533,7 +533,7 @@
 ')
 
 optional_policy(`
-	unconfined_stub_role()
+#	unconfined_stub_role()
 	unconfined_domain(container_runtime_t)
 	unconfined_run_to(container_runtime_t, container_runtime_exec_t)
 	role_transition unconfined_r container_runtime_exec_t system_r;
@@ -544,19 +544,19 @@
 
 optional_policy(`
 	virt_read_config(container_runtime_domain)
-	virt_exec(container_runtime_domain)
+#	virt_exec(container_runtime_domain)
 	virt_stream_connect(container_runtime_domain)
-	virt_stream_connect_sandbox(container_runtime_domain)
-	virt_exec_sandbox_files(container_runtime_domain)
-	virt_manage_sandbox_files(container_runtime_domain)
-	virt_relabel_sandbox_filesystem(container_runtime_domain)
+#	virt_stream_connect_sandbox(container_runtime_domain)
+#	virt_exec_sandbox_files(container_runtime_domain)
+#	virt_manage_sandbox_files(container_runtime_domain)
+#	virt_relabel_sandbox_filesystem(container_runtime_domain)
 	# for lxc
-	virt_mounton_sandbox_file(container_runtime_domain)
+#	virt_mounton_sandbox_file(container_runtime_domain)
 #	virt_attach_sandbox_tun_iface(container_runtime_domain)
 	allow container_runtime_domain container_domain:tun_socket relabelfrom;
-	virt_sandbox_entrypoint(container_runtime_domain)
-	virt_stub_lxc()
-	allow container_runtime_domain virtd_lxc_t:unix_stream_socket { rw_stream_socket_perms connectto };
+#	virt_sandbox_entrypoint(container_runtime_domain)
+#	virt_stub_lxc()
+#	allow container_runtime_domain virtd_lxc_t:unix_stream_socket { rw_stream_socket_perms connectto };
 
 ')
 
@@ -587,15 +587,15 @@
 init_dbus_chat(spc_t)
 
 optional_policy(`
-	systemd_dbus_chat_machined(spc_t)
+#	systemd_dbus_chat_machined(spc_t)
 	systemd_dbus_chat_logind(spc_t)
 ')
 
-optional_policy(`
-	dbus_chat_system_bus(spc_t)
-	dbus_chat_session_bus(spc_t)
-	dnsmasq_dbus_chat(spc_t)
-')
+# optional_policy(`
+# 	dbus_chat_system_bus(spc_t)
+# 	dbus_chat_session_bus(spc_t)
+# 	dnsmasq_dbus_chat(spc_t)
+# ')
 
 optional_policy(`
 	unconfined_domain_noaudit(spc_t)
@@ -603,12 +603,12 @@
 ')
 
 optional_policy(`
-	virt_transition_svirt_sandbox(spc_t, system_r)
-	virt_sandbox_entrypoint(spc_t)
-	virt_sandbox_domtrans(container_runtime_domain, spc_t)
-	virt_transition_svirt(spc_t, system_r)
-	virt_sandbox_entrypoint(container_file_t)
-	virt_sandbox_entrypoint(container_ro_file_t)
+#	virt_transition_svirt_sandbox(spc_t, system_r)
+#	virt_sandbox_entrypoint(spc_t)
+#	virt_sandbox_domtrans(container_runtime_domain, spc_t)
+#	virt_transition_svirt(spc_t, system_r)
+#	virt_sandbox_entrypoint(container_file_t)
+#	virt_sandbox_entrypoint(container_ro_file_t)
 
 	gen_require(`
 		attribute virt_domain;
@@ -695,8 +695,8 @@
 allow container_domain self:unix_stream_socket create_stream_socket_perms;
 dontaudit container_domain self:capability2  block_suspend ;
 allow container_domain self:unix_stream_socket { sendto create_stream_socket_perms };
-fs_rw_onload_sockets(container_domain)
-fs_fusefs_entrypoint(container_domain)
+# fs_rw_onload_sockets(container_domain)
+# fs_fusefs_entrypoint(container_domain)
 
 manage_files_pattern(container_domain, container_file_t, container_file_t)
 exec_files_pattern(container_domain, container_file_t, container_file_t)
@@ -715,9 +715,9 @@
 container_exec_share_files(container_domain)
 container_use_ptys(container_domain)
 container_spc_stream_connect(container_domain)
-fs_dontaudit_remount_tmpfs(container_domain)
-dev_dontaudit_mounton_sysfs(container_domain)
-dev_dontaudit_mounton_sysfs(container_domain)
+# fs_dontaudit_remount_tmpfs(container_domain)
+# dev_dontaudit_mounton_sysfs(container_domain)
+# dev_dontaudit_mounton_sysfs(container_domain)
 
 dontaudit container_domain container_runtime_tmpfs_t:dir read;
 dev_getattr_mtrr_dev(container_domain)
@@ -746,45 +746,45 @@
 kernel_rw_net_sysctls(container_domain)
 kernel_rw_unix_sysctls(container_domain)
 kernel_dontaudit_search_kernel_sysctl(container_domain)
-kernel_dontaudit_access_check_proc(container_domain)
-kernel_dontaudit_setattr_proc_files(container_domain)
+# kernel_dontaudit_access_check_proc(container_domain)
+# kernel_dontaudit_setattr_proc_files(container_domain)
 kernel_dontaudit_setattr_proc_dirs(container_domain)
-kernel_dontaudit_write_usermodehelper_state(container_domain)
+# kernel_dontaudit_write_usermodehelper_state(container_domain)
 kernel_read_irq_sysctls(container_domain)
 kernel_get_sysvipc_info(container_domain)
 
 fs_getattr_all_fs(container_domain)
 fs_list_inotifyfs(container_domain)
-fs_rw_inherited_tmpfs_files(container_domain)
-fs_read_hugetlbfs_files(container_domain)
+# fs_rw_inherited_tmpfs_files(container_domain)
+# fs_read_hugetlbfs_files(container_domain)
 fs_read_tmpfs_symlinks(container_domain)
 fs_search_tmpfs(container_domain)
 fs_rw_hugetlbfs_files(container_domain)
-fs_exec_hugetlbfs_files(container_domain)
-fs_dontaudit_getattr_all_dirs(container_domain)
+# fs_exec_hugetlbfs_files(container_domain)
+# fs_dontaudit_getattr_all_dirs(container_domain)
 fs_dontaudit_getattr_all_files(container_domain)
 
-term_use_all_inherited_terms(container_domain)
+# term_use_all_inherited_terms(container_domain)
 
 userdom_use_user_ptys(container_domain)
-userdom_rw_inherited_user_pipes(container_domain)
+# userdom_rw_inherited_user_pipes(container_domain)
 
-domain_dontaudit_link_all_domains_keyrings(container_domain)
-domain_dontaudit_search_all_domains_keyrings(container_domain)
+# domain_dontaudit_link_all_domains_keyrings(container_domain)
+# domain_dontaudit_search_all_domains_keyrings(container_domain)
 
-virt_stub_svirt_sandbox_file()
-virt_sandbox_net_domain(container_t)
+# virt_stub_svirt_sandbox_file()
+# virt_sandbox_net_domain(container_t)
 
 logging_send_syslog_msg(container_t)
 
-fs_noxattr_type(container_file_t)
+# fs_noxattr_type(container_file_t)
 # fs_associate_cgroupfs(container_file_t)
 gen_require(`
 	type cgroup_t;
 ')
 
 dev_read_sysfs(container_domain)
-dev_read_mtrr(container_domain)
+# dev_read_mtrr(container_domain)
 dev_mounton_sysfs(container_t)
 
 fs_mounton_cgroup(container_t)
@@ -796,8 +796,8 @@
 files_read_kernel_modules(container_domain)
 
 allow container_file_t cgroup_t:filesystem associate;
-term_pty(container_file_t)
-logging_log_file(container_file_t)
+# term_pty(container_file_t)
+# logging_log_file(container_file_t)
 tunable_policy(`virt_sandbox_use_sys_admin',`
 	allow container_t self:capability sys_admin;
 	allow container_t self:cap_userns sys_admin;
@@ -808,14 +808,14 @@
 
 corenet_unconfined(container_t)
 
-optional_policy(`
-	virt_default_capabilities(container_t)
-')
+# optional_policy(`
+# 	virt_default_capabilities(container_t)
+# ')
 kernel_rw_rpc_sysctls(container_domain)
 kernel_rw_net_sysctls(container_domain)
 kernel_read_messages(container_t)
 kernel_read_network_state(container_domain)
-kernel_dontaudit_write_proc_files(container_domain)
+# kernel_dontaudit_write_proc_files(container_domain)
 
 # Container Net Domain
 corenet_tcp_bind_generic_node(container_net_domain)
@@ -839,8 +839,8 @@
 allow container_net_domain self:netlink_xfrm_socket create_netlink_socket_perms;
 
 
-kernel_unlabeled_domtrans(container_runtime_domain, spc_t)
-kernel_unlabeled_entry_type(spc_t)
+# kernel_unlabeled_domtrans(container_runtime_domain, spc_t)
+# kernel_unlabeled_entry_type(spc_t)
 allow container_runtime_domain unlabeled_t:key manage_key_perms;
 #kernel_dontaudit_write_usermodehelper_state(container_t)
 gen_require(`
@@ -879,9 +879,9 @@
 
 fs_manage_fusefs_dirs(container_domain)
 fs_manage_fusefs_files(container_domain)
-fs_manage_fusefs_symlinks(container_domain)
-fs_manage_fusefs_named_sockets(container_domain)
-fs_manage_fusefs_named_pipes(container_domain)
+# fs_manage_fusefs_symlinks(container_domain)
+# fs_manage_fusefs_named_sockets(container_domain)
+# fs_manage_fusefs_named_pipes(container_domain)
 fs_exec_fusefs_files(container_domain)
 fs_unmount_xattr_fs(container_domain)
 fs_mount_fusefs(container_domain)
@@ -933,16 +933,16 @@
 container_append_file(iptables_t)
 
 optional_policy(`
-	unconfined_stub_role()
+#	unconfined_stub_role()
 	gen_require(`
 		type unconfined_service_t;
 		type unconfined_service_exec_t;
 	')
 
-	virt_transition_svirt_sandbox(unconfined_service_t, system_r)
+#	virt_transition_svirt_sandbox(unconfined_service_t, system_r)
 	container_filetrans_named_content(unconfined_service_t)
 	container_runtime_domtrans(unconfined_service_t)
-	role_transition unconfined_r unconfined_service_exec_t system_r;
+#	role_transition unconfined_r unconfined_service_exec_t system_r;
 	allow container_runtime_domain unconfined_service_t:fifo_file setattr;
 	allow unconfined_service_t container_domain:process dyntransition;
 	allow unconfined_service_t unlabeled_t:key manage_key_perms;
@@ -963,8 +963,8 @@
 #
 container_domain_template(container_userns)
 
-typeattribute  container_userns_t sandbox_net_domain;
-dev_mount_sysfs_fs(container_userns_t)
+# typeattribute  container_userns_t sandbox_net_domain;
+# dev_mount_sysfs_fs(container_userns_t)
 dev_mounton_sysfs(container_userns_t)
 
 fs_mount_tmpfs(container_userns_t)
@@ -978,7 +978,7 @@
 
 term_use_generic_ptys(container_userns_t)
 term_setattr_generic_ptys(container_userns_t)
-term_mount_pty_fs(container_userns_t)
+# term_mount_pty_fs(container_userns_t)
 
 allow container_userns_t self:capability ~{ sys_module };
 allow container_userns_t self:capability2 ~{ mac_override mac_admin };
@@ -1078,9 +1078,9 @@
 
 container_stream_connect(container_kvm_t)
 
-dev_rw_inherited_vhost(container_kvm_t)
+# dev_rw_inherited_vhost(container_kvm_t)
 
-corenet_rw_inherited_tun_tap_dev(container_kvm_t)
+# corenet_rw_inherited_tun_tap_dev(container_kvm_t)
 corecmd_exec_shell(container_kvm_t)
 corecmd_exec_bin(container_kvm_t)
 corecmd_bin_entry_type(container_kvm_t)
@@ -1088,9 +1088,9 @@
 # virtiofs causes these AVC messages.
 kernel_mount_proc(container_kvm_t)
 kernel_mounton_proc(container_kvm_t)
-files_mounton_rootfs(container_kvm_t)
+# files_mounton_rootfs(container_kvm_t)
 
-auth_read_passwd(container_kvm_t)
+# auth_read_passwd(container_kvm_t)
 
 optional_policy(`
 	qemu_entry_type(container_kvm_t)
@@ -1118,9 +1118,9 @@
 
 allow container_init_t proc_t:filesystem remount;
 
-optional_policy(`
-	virt_default_capabilities(container_init_t)
-')
+# optional_policy(`
+# 	virt_default_capabilities(container_init_t)
+# ')
 
 tunable_policy(`virt_sandbox_use_sys_admin',`
 	allow container_init_t self:capability sys_admin;
